lib/samples/style_graphics_with_renderer/README.md

```diff
@@ -0,0 +1,42 @@
+# Style graphics with renderer
+
+A renderer allows you to change the style of all graphics in a graphics overlay by referencing a single symbol style. A renderer will only affect graphics that do not specify their own symbol style.
+
+![Image of add graphics with renderer](style_graphics_with_renderer.dart)
+
+## Use case
+
+A renderer allows you to change the style of all graphics in an overlay by only changing one copy of the symbol. For example, a user may wish to display a number of graphics on a map of parkland which represent trees, all sharing a common symbol.
+
+## How to use the sample
+
+Pan and zoom on the map to view graphics for points, lines, and polygons (including polygons with curved segments), which are stylized using renderers.
+
+## How it works
+
+1. Create a `GraphicsOverlay` and add it to the `MapView`.
+2. Create a `Graphic`, specifying only a `Geometry`.
+3. Create a single `Symbol` such as a `SimpleMarkerSymbol`.
+4. Create a renderer with `SimpleRenderer(symbol:)`, passing in a `Symbol`.
```

Comment:
We would just call this method "SimpleRenderer()" (and Symbol => ArcGISSymbol)

Reply:
Changed It